subsys/nrf_security/src/drivers/cracen/cracenpsa/src/common.h

```diff
@@ -301,3 +315,35 @@ int cracen_get_rnd_in_range(const uint8_t *n, size_t nsz, uint8_t *out);
  */
 psa_status_t cracen_ecc_reduce_p256(const uint8_t *input, size_t input_size, uint8_t *output,
 				    size_t output_size);
+int cracen_memdiff(const uint8_t *a, const uint8_t *b, size_t sz);
+
+/** Modular exponentiation (base^key mod n).
+ *
+ * This function is used by both the sign and the verify functions. Note: if the
+ * base is greater than the modulus, SilexPK will return the SX_ERR_OUT_OF_RANGE
+ * status code.
+ */
+int cracen_rsa_modexp(struct sx_pk_acq_req *pkreq, struct sx_pk_slot *inputs,
+		      struct cracen_rsa_key *rsa_key, uint8_t *base, size_t basez, int *sizes);
+
+
+/** Initialize an RSA key consisting of modulus and exponent. */
+#define CRACEN_KEY_INIT_RSA(mod, expon)                                                            \
+	(struct cracen_rsa_key)                                                                    \
+	{                                                                                          \
+		.cmd = SX_PK_CMD_MOD_EXP, .slotmask = (1 | (1 << 2)), .dataidx = 1, {mod, expon},  \
```

Comment:
Would be more readable with one assignment per line (not including {mod, expon})